src/styles/blocks/calendar.scss

```
@@ -0,0 +1,45 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $item-gap;
+  width: ($item-size * 8) + ($item-gap * 6);
+  margin: 0 auto;
+  padding-top: 50px;
+
+  @each $day, $index in $days {
+    &--start-day--#{$day} &__day:first-child {
+      margin-left: calc((#{$item-gap} + #{$item-size}) * #{$index});
+    }
+  }
+
+  @for $monthDay from 28 through 31 {
+    $monthDayNew: $monthDay + 1;
+
+    &--month-length--#{$monthDay} :nth-child(n + #{$monthDayNew}) {
+      display: none;
```

You can do simple calculation with variables inside its envelop #{}
```suggestion
    &--month-length--#{$monthDay} :nth-child(n + #{$monthDay + 1}) {
      display: none;
```